src/Creator/RefundUnitsCommandCreatorInterface.php

```diff
@@ -13,10 +13,9 @@
 
 namespace Sylius\RefundPlugin\Creator;
 
-use Sylius\RefundPlugin\Command\RefundUnits;
 use Symfony\Component\HttpFoundation\Request;
 
 interface RefundUnitsCommandCreatorInterface
 {
-    public function fromRequest(Request $request): RefundUnits;
+    public function fromRequest(Request $request): object;
```

Comment:
Because of that change, `RefundUnitsCommandCreatorInterface` might be eg. `RequestCommandCreatorInterface` as it might be implemented for any command. WDYT?